experiments/20161122-pacbio/barcode-insert-primers.md

```diff
@@ -0,0 +1,15 @@
+## Primers for inserting barcode
+
+Since the ORF of PB2 takes up the majority of the sequence and since the UTR needs to be intact for polymerase function, the barcode will be inserted in the pCI backbone.
+
+**Option 1: Before PB2**
+Forward: CAGCTCTTAAGGCTAGAGTACTTAATNNNNNNNNACGACTCACTATAGGCTAGCCTCGAG
+Reverse: CTCGAGGCTAGCCTATAGTGAGTCGTNNNNNNNNATTAAGTACTCTAGCCTTAAGAGCTG
+Tm: 77.2C, 60-mer total
+
+**Option 2: After PB2**
```

Comment:
Likewise here for indicating where the first N of N<sub>8</sub> will be, e.g.: "25 n.t. downstream of PB2 end."